src/models/GitHub.js

```diff
@@ -1,157 +1,158 @@
+
+      return response.json().then(result => {
+        if (result.errors) {
+          throw new Error(`GraphQL Error: ${result.errors.map(e => e.message)}`)
+        }
+
+        return result.data
+      })
+    })
   }
 
   getHeaders() {
     if (!this.token) {
       this.token = GitHubAuth.getToken()
     }
     return {
-      Accept: 'application/vnd.github.v3+json',
-      Authorization: `token ${this.token}`,
+      Authorization: `bearer ${this.token}`,
     }
   }
 
-  get(path) {
-    const url = this.getFullUrl(path)
-    const opts = { headers: this.getHeaders() }
-    return super.get(url, opts)
-  }
-
-  getFullUrl(relativeOrAbsoluteUrl) {
-    if (relativeOrAbsoluteUrl.indexOf('http') !== 0) {
-      return `${Config.githubApiUrl}/${relativeOrAbsoluteUrl}`
-    }
-    return relativeOrAbsoluteUrl
-  }
-
-  getNextUrl(headers) {
-    const link = headers.get('Link')
-    if (!link) {
-      return null
+  transformEdgeToTask(edge) {
+    const node = edge.node
+    const type = node.type === 'PullRequest' ? 'pull' : 'issue'
+    return {
+      storageKey: `${type}-${node.id}`,
+      id: node.id,
+      type: node.type,
+      title: node.title,
+      body: node.body,
+      state: node.state,
+      createdAt: node.createdAt,
+      updatedAt: node.updatedAt,
+      isPullRequest: node.type === 'PullRequest',
+      url: node.url,
+      number: node.number,
+      repository: `${node.repository.owner.login}/${node.repository.name}`,
+      repositoryOwner: node.repository.owner.login,
+      repositoryOwnerUrl: node.repository.owner.url,
+      repositoryOwnerAvatar: node.repository.owner.avatarURL,
+      user: node.author.login,
+      userUrl: node.author.url,
+      userAvatar: node.author.avatarURL,
+      comments: node.comments ? node.comments.totalCount : 0,
     }
-    return this.getNextUrlFromLink(link)
   }
 
-  // Sample input:
-  // Link: <https://api.github.com/search/code?q=addClass+user%3Amozilla&page=15>; rel="next",
-  // <https://api.github.com/search/code?q=addClass+user%3Amozilla&page=34>; rel="last",
-  // <https://api.github.com/search/code?q=addClass+user%3Amozilla&page=1>; rel="first",
-  // <https://api.github.com/search/code?q=addClass+user%3Amozilla&page=13>; rel="prev"
-  //
-  // Sample output:
-  // https://api.github.com/search/code?q=addClass+user%3Amozilla&page=15
-  getNextUrlFromLink(link) {
-    const urlsAndRels = link.split(',')
-    let nextUrl
-    urlsAndRels.forEach(str => {
-      const urlAndRel = str.trim().split('; ')
-      if (urlAndRel[1] === 'rel="next"') {
-        const urlInBrackets = urlAndRel[0]
-        nextUrl = urlInBrackets.slice(1, urlInBrackets.length - 1)
-        return
+  taskQuery() {
+    return `query($search: String!, $endCursor: String) {
+      search(first: 30, query: $search, after: $endCursor, type: ISSUE) {
+        pageInfo {
+         endCursor,
+          hasNextPage
+        }
+        edges {
+          node {
+            ... on Issueish {
```

Comment:
Was there documentation you were following about the types available? If so, maybe link to it in a comment here but also in the readme.